check-dep-specs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
lets create a rule in rules_docker contrib to run the docker command. Rule takes in a list of yaml runs a separate shell action for each to run the check